aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -132,6 +133,22 @@ TORCH_META_FUNC(_convert_indices_from_csr_to_coo)
   set_output_raw_strided(0, {2, col_indices.numel()}, {}, options, {});
 }
 
+TORCH_META_FUNC(_compressed_to_batched_compressed_indices)
+(const Tensor& compressed_indices,
+ const int64_t n_batch,
+ const bool out_int32) {
+  TORCH_CHECK(
+      compressed_indices.dim() == 1,
+      "Compressed indices is supposed to be a vector");
```

Comment:
if you're in here doing other things, probably worth adding the `compressed_indices.dim()` we actually got